### ISSUE TITLE
Bash completion does not work in google-cloud-sdk Debian package

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#280]
```
After installing the google-cloud-sdk Debian package, as documented on 
<https://cloud.google.com/sdk/#debubu>, on either Debian or Ubuntu systems, the 
Bash tab-completion for the SDK commands returns files rather than the 
programmable completion options provided in 
/etc/bash_completion.d/google-cloud-sdk/completion.bash.inc.

This is because bash-completion does not search /etc/bash_completion.d 
recursively, as seen in the source at 
<https://anonscm.debian.org/cgit/bash-completion/bash-completion.git/tree/bash_c
ompletion?h=2.1#n1963>, and therefore skips the completion.bash.inc file in the 
google-cloud-sdk subdirectory.  This can be fixed by moving the file into 
/etc/bash_completion.d or symlinking to it.

While fixing this issue, you may want to consider installing the file using 
dh_bash-completion in the Debian packages, which will place the file in 
/usr/share/bash-completion/completions which is now preferred according to the 
bash-completion maintainers 
<https://anonscm.debian.org/cgit/bash-completion/debian.git/tree/debian/README.D
ebian>.

Thanks,
Kevin
```

Original issue reported on code.google.com by `ke...@kevinlocke.name` on 2 Oct 2015 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this! We'll take a look at our .deb packaging process.
```

Original comment by `z...@google.com` on 5 Oct 2015 at 8:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the delay. A fix for this should go out in version 0.9.89.
```

Original comment by `stephe...@google.com` on 23 Nov 2015 at 10:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Not a problem.  Thanks for the update and the fix!  I'll test once it hits the 
repositories and re-open if I encounter a problem.  Thanks again.
```

Original comment by `ke...@kevinlocke.name` on 23 Nov 2015 at 11:21
